src/index.ts

```diff
@@ -67,10 +80,24 @@ export async function readResources(
       "{language}",
       language
     );
-    const json = JSON.parse((await args.$fs.readFile(resourcePath, "utf-8")) as string)
-    // reading the json, and flattening it to avoid nested keys.
-    const flatJson = flatten(json) as Record<string, string>;
-    result.push(parseResource(flatJson, language));
+    if((await args.$fs.stat(`${resourcePath.replace("/*.json", "")}`)).isFile()) {
```

Comment:
We won't have access to `stat` anymore with the upcoming https://github.com/inlang/inlang/pull/559 :(

Any way we can achieve the same effect with only `readFile`, `mkdir`, `writeFile` and `readdir`? See https://github.com/inlang/inlang/pull/559/files#diff-e0a2f5d5dc03af58ffcd338bb5eeeaad78aec678b17e294c25f0e6cfa1079b02R12

Reply:
I replaced it with a try if the read fails it switches over to the catch statement and does the other.